bipen/src/main/java/com/lftechnology/phpjava/assignmenttwo/MainRunner.java

```diff
@@ -0,0 +1,53 @@
+package com.lftechnology.phpjava.assignmenttwo;
+
+import java.util.Scanner;
+
+/**
+ * Main Class to run the application with various available options.
+ * 
+ * @author Bipen Chhetri <bipenchhetri@lftechnology.com
+ * 
+ */
+public class MainRunner {
+
+    public static MainRunner runner;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static runner" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1444)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make runner a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)